contrib/platform/src/com/sun/jna/platform/win32/Winspool.java

```diff
@@ -190,6 +195,16 @@ public interface Winspool extends StdCallLibrary {
     public static final short PRINTER_NOTIFY_FIELD_FRIENDLY_NAME = 0x1B;
     public static final short PRINTER_NOTIFY_FIELD_BRANCH_OFFICE_PRINTING = 0x1C;
 
+    public static final short JOB_CONTROL_PAUSE = 0x01;
+    public static final short JOB_CONTROL_RESUME = 0x02;
+    public static final short JOB_CONTROL_CANCEL = 0x03;
+    public static final short JOB_CONTROL_RESTART = 0x04;
+    public static final short JOB_CONTROL_DELETE = 0x05;
+    public static final short JOB_CONTROL_SENT_TO_PRINTER = 0x06;
+    public static final short JOB_CONTROL_LAST_PAGE_EJECTED = 0x07;
+    public static final short JOB_CONTROL_RETAIN = 0x08;
+    public static final short JOB_CONTROL_RELEASE = 0x09;
```

Comment:
This is only ever used in `int` context. I don't see a reason to declare it different.